### ISSUE TITLE
CORS Issues with Youtube Embed

### ISSUE THREAD

**Issue by joebentaylor1995** [CookPete/react-player#875]
My code:
```
<ReactPlayer
    url={block.youtubeEmbed}
    className='react-player'
    width='100%'
    height='100%'
/>
```

`block.youtubeEmbed` is just pulling the youtube URL

Console Error:
```
Failed to execute 'postMessage' on 'DOMWindow': The target origin provided ('https://www.youtube.com') does not match the recipient window's origin ('http://localhost:8000').

Access to XMLHttpRequest at 'https://googleads.g.doubleclick.net/pagead/id' from origin 'https://www.youtube.com' has been blocked by CORS policy: The value of the 'Access-Control-Allow-Origin' header in the response must not be the wildcard '*' when the request's credentials mode is 'include'. The credentials mode of requests initiated by the XMLHttpRequest is controlled by the withCredentials attribute.

GET https://googleads.g.doubleclick.net/pagead/id net::ERR_FAILED
```

**Comment by joebentaylor1995** [comment]
Ive managed to get rid of the first line error by making sure the video is put in as https:// but the other two I cant figure out

**Comment by danihazler** [comment]
I'm getting your same error 

```  Failed to execute 'postMessage' on 'DOMWindow': The target origin provided ('https://www.youtube.com') does not match the recipient window's origin ('http://localhost:3000'). ```

even if the link is https://

**Comment by CookPete** [comment]
This feels like a duplicate of https://github.com/CookPete/react-player/issues/508#issuecomment-454426112. 

If this is just weirdness with the youtube player API I'm not sure what more this library can do to prevent it.

**Comment by danihazler** [comment]
Thanks for your attention!  @CookPete
